server/controller/Users/UserQueries.js

```diff
@@ -13,7 +15,7 @@ module.exports = {
     register: (data, callBack) => {
         dbConfig.query(
             `insert into users(username, password, userTypeID) 
-                        values(?,?,1)`,
+                        values(?,?,rand())`,
```

Comment:
Please fix this, we only  use 0 and 1 for userTypeID, so we won't insert that randomly, just insert 0 from now for user. And please make your variable is a constant and reference to this query